### PR TITLE
added changed js, increased max-results etc

### PR DESCRIPTION
**Pull request by priya-sudarshanam**
@rbran100
following changes are put in:
a)js changed to include the url
b)max-results increased to 10
c)added braces to the views
d)removed /home from the status bar

### CODE REVIEW

File: ossap_stats/ossap_stats_credentials.inc [hunk 1]
```diff
@@ -77,13 +77,13 @@ function ossap_stats_design_output($type){
         $sort = $sort_visits;
         $metrics = $metrics_visits;
         $dimensions = 'ga:hostname';
-        $max_results = 6;
+        $max_results = 11;
```
Comment (rbran100) [inline ossap_stats/ossap_stats_credentials.inc:80]: Why 11, out of curiosity?

File: ossap_stats/ossap_stats.module [hunk 2]
```diff
@@ -230,13 +231,15 @@ function ossap_stats_rt_visitors($output_json = FALSE){
      
      $data = $results['rows'][0][0];
      
+     variable_set('rt_visitors', $data);
```
Comment (rbran100) [inline ossap_stats/ossap_stats.module:234]: Nice, good idea.

File: ossap_stats/ossap_stats_credentials.inc (hunk 1 above)
Comment (priya-sudarshanam) [inline ossap_stats/ossap_stats_credentials.inc:80]: well the results that are coming out needs to filter out the portals. for sites, the top most site is the portal scholar.harvard.edu. Removing this creates uneveness so i have to increase it to 100 balance out the results..Hope it makes sense

Comment (priya-sudarshanam) [inline ossap_stats/ossap_stats_credentials.inc:80]: sorry increase it to 10...my bad

**Pull request merged**